<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>195</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="provinces">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="citys">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择城市</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>59</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>276</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelImage1">
          <property name="text">
           <string>image1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelImage2">
          <property name="text">
           <string>image2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelTemperature">
        <property name="text">
         <string>20℃/27℃</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWeather">
        <property name="text">
         <string> 小雨转阴</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWind">
        <property name="text">
         <string>西风4-5级转西北风4-5级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelImage3">
          <property name="text">
           <string>image3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelImage4">
          <property name="text">
           <string>image4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelTemperature_2">
        <property name="text">
         <string>20℃/27℃</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWeather_2">
        <property name="text">
         <string> 小雨转阴</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWind_2">
        <property name="text">
         <string>西风4-5级转西北风4-5级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>provinces</zorder>
  <zorder>citys</zorder>
  <zorder>labelCity</zorder>
  <zorder>findButton</zorder>
  <zorder>labelImage1</zorder>
  <zorder>labelTemperature</zorder>
  <zorder>labelWeather</zorder>
  <zorder>labelWind</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>labelLogo</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
